programmers/난이도별/level01.정수_내림차순으로_배치하기/HyeonJeong.py

```diff
@@ -0,0 +1,7 @@
+def solution(n):
+    answer = list(str(n) #answer = [c for c in str(n)]
+    answer.sort(reverse = True)
+    return int("".join(answer))
```

Comment:
```suggestion
    return int("".join(sorted(list(str(n), reverse = True)))
```

이렇게 `sort`를 이용하면 원본 배열이 변경되고, `sorted`를 이용하면 원본 배열을 정렬한 새로운 값을 반환합니다.

Reply:
두개의 차이 알려주셔서 감사합니다!